src/test/java/net/starschema/clouddb/jdbc/BQScrollableResultSetFunctionTest.java

```diff
@@ -745,4 +747,31 @@ public void testStatelessQuery() throws SQLException {
     Assertions.assertThat(bqResultSet.getJobId()).isNull();
     Assertions.assertThat(bqResultSet.getQueryId()).contains("!");
   }
+
+  @Override
+  protected Statement createStatementForCommonTests(Connection connection) throws SQLException {
+    return connection.createStatement(
+        ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);
+  }
+
+  @Override
+  @Test
+  @Ignore(
+      "Contradiction between BQSCrollableResultSet and BQForwardOnlyResultSet "
+          + "dates and times: b/317107706")
+  public void testGetTimeByLabel() throws SQLException {}
+
+  @Override
+  @Test
+  @Ignore(
+      "Contradiction between BQSCrollableResultSet and BQForwardOnlyResultSet "
+          + "dates and times: b/317107706")
+  public void testGetDateByLabel() throws SQLException {}
+
+  @Override
+  @Test
+  @Ignore(
+      "Contradiction between BQSCrollableResultSet and BQForwardOnlyResultSet "
+          + "dates and times: b/317107706")
```

Comment:
Could you double check whether bug ID is confidential? A brief search says no but just to be safe..